.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -50,5 +62,9 @@ Steps to reproduce the behavior:
 
 **Additional context**
 <!--
-(Add any other context about the problem here)
+Add any other context about the problem here. E.g.: 
+1. Additional checks done with outcome, if that can help understand the issue better or pinpoint the root cause
+2. Related issues 
+3. Logs (if available)
+4. Impact on the end user in general.
```

Comment:
Is there any non-obvious/serious behaviour/effect on the end-user with this issue?

Reply:
Done